test/test_operations.py

```diff
@@ -2118,6 +2118,59 @@ def fn(inp, s):
     self.assertEqual(out, Xout.cpu())
     self.assertEqual("f16", torch_xla._XLAC._get_xla_tensor_shape_type(Xout))
 
+  # We skip TPU for 2 reasons:
+  #   1. upsample_bilinear on f64 tensors doesn't work on TPUs
```

Comment:
will f64 works prior to your change on TPU? I don't want this to become a regression.

Reply:
My change doesn't affect TPU. The lowering for `upsample_bilinear` is different for CUDA and TPU. This change only affects non-TPU and non-Neuron devices.